BaseTools/Conf/XMLSchema/DistributionPackage.xsd

```diff
@@ -330,7 +330,7 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
               having an error
             </xs:documentation>
             <xs:documentation xml:lang="en-us">
-              #3 Allow for inclusion of ARM and future architecture
+              #3 Allow for inclusion of  and future architecture

```

Comment:
No idea what this file does but this change looks slightly dodgy. Care to fix this, or explain why changing it like this is appropriate?